### PR TITLE
News just has things that are new. Status has things that are old

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: content/en/project/status/_index.md [hunk 1]
```diff
@@ -7,18 +7,26 @@ weight: 50
 ## Annual Reports
 
 The Medley team has produced annual reports documenting their major achievements.
```
Comment (stumbo) [inline content/en/project/status/_index.md:9]: If you want this on a separate line from the **Sneak Peak:** sentence add a couple spaces to the end.  If you would also like a paragraph break and a new line with 2 spaces in it.

File: content/en/project/status/_index.md [hunk 2]
```diff
@@ -7,18 +7,26 @@ weight: 50
...
+Sneak Peek: We are working on our [2023 Medley Interlisp Annual Report](https://docs.google.com/document/d/1yL1WeYvgu5JstWyjD3fWl1mDZv8kIr_U3_1onSWvUko/edit?usp=sharing). Still a work in progress -- don't distribute until we announce it ready.
+## 12/2022
```
Comment (stumbo) [inline content/en/project/status/_index.md:11]: ### `### 12/2022` to get the right size header.

File: content/en/project/status/_index.md [hunk 3]
```diff
@@ -7,18 +7,26 @@ weight: 50
...
 <iframe width="560" height="315" src="https://www.youtube.com/embed/N1MobfEaoWY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-## Closed Issues
+Drafts and final versions of slide decks can be found [here](https://drive.google.com/drive/folders/1wQZ9F-doxx5gsGaQ7gKtOa5AObJalNJ9?usp=sharing)
+
+
+### Closed Issues
 
 The [issues closed](https://github.com/Interlisp/medley/issues?q=is%3Aissue+is%3Aclosed) by the team can be found in our GitHub repository.
+<!-- is there a way to ask for "closed in the last N days"?  -->
```
Comment (stumbo) [inline content/en/project/status/_index.md:32]: Issues closed in 2022:  [https://github.com/Interlisp/medley/issues?page=6&q=is%3Aissue+closed%3A%3E2022-12-31](https://github.com/Interlisp/medley/issues?page=6&q=is%3Aissue+closed%3A%3E2022-12-31)

I don't see a way of getting the last n days from the day the query is run.

**Review by stumbo — CHANGES_REQUESTED** [review]
Minor clean up requests.

**Comment by stumbo** [comment]
Do you want to close this one and mark it as superseded by #194 ?

**Comment by masinter** [comment]
superseded by 195

**Pull request closed without merging**